source/print-commit-log.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
Same, extracted it, because it's used in `detect-version.js`.

Reply:
I also removed side effects from this function, previously it was responsible for generating release notes as well.

Reply:
Why does this print the commit log? That seems a bit out-of-scope to me. Let's change this to only return the commit log, and then we can `console.log` the output when needed.